spark.js

```diff
@@ -195,12 +164,6 @@ Spark.readable('__initialise', [function initialise() {
   // We've received new data from our client, decode and emit it.
   //
   ultron.on('incoming::data', function message(raw) {
-    //
```

Comment:
This only reset the timer. The client will keep sending its pings to which the server responds.